messaging.rst

```diff
@@ -276,6 +276,148 @@ This should match `the Secret message in encoding/messages file <https://github.
 |                      |                        | is the hash of the whole message                           |
 +----------------------+------------------------+------------------------------------------------------------+
 
+.. _withdraw-request-message:
+
+Withdraw Request
+-------------
+
+Message used by the a channel participant node to request the other participant signature on a new increased ``total_withdraw`` value.
+
+Preconditions
+^^^^^^^^^^^^^
+
+- The channel for which the withdraw is requested should be open.
+- The ``total_withdraw`` value should only ever increase.
+- The participant's channel unlocked balance should be larger or equal to ``withdraw_amount``,
+  which is calculated using ``new_total_withdraw - previous_total_withdraw``.
+- The new total_withdraw value should not cause an underflow or overflow.
+- The message should be sent by one of the channel participants.
```

Comment:
As this is a require in the contracts, I'd use must here.

Reply:
done